docs/book/src/reference/envtest.md

```diff
@@ -97,12 +97,12 @@ You can use environment variables and/or flags to specify the `kubectl`,`api-ser
 ### Environment Variables
```

Comment:
Under Writing test, at the bottom of the section see: https://github.com/kubernetes-sigs/kubebuilder/blob/7487261ff6bf915b936f47684b2ffe1b23e2d689/docs/book/src/reference/envtest.md#writing-tests

We can add the following note like:

```
<aside class="note">
<h1>Examples</h1>

You can use the plugin [DeployImage](https://book.kubebuilder.io/plugins/deploy-image-plugin-v1-alpha.html) to check examples. This plugin allows users to scaffold API/Controllers to deploy and manage an Operand (image) on the cluster following the guidelines and best practices. It abstracts the complexities of achieving this goal while allowing users to customize the generated code.

Therefore, you can check that a test using ENV TEST will be generated for the controller which has the purpose to ensure that the Deployment is created succeffully. You can see an example of its code implementation under the `testdata` directory with the [DeployImage](https://book.kubebuilder.io/plugins/deploy-image-plugin-v1-alpha.html) samples [here](https://github.com/kubernetes-sigs/kubebuilder/blob/v3.7.0/testdata/project-v3-with-deploy-image/controllers/busybox_controller_test.go).

</aside>
```